packages/gatsby-theme-docs/src/components/content-pagination.js

```diff
@@ -15,7 +15,7 @@ const trimTrailingSlash = url => url.replace(/(\/?)$/, '');
 
 const isMatching = (a, b) => trimTrailingSlash(a) === trimTrailingSlash(b);
 
-const Container = styled.div`
+const Container = styled.nav`
```

Comment:
Please assign a `aria-label` to the `<Container>` element to differentiate it from other navigation elements

> If a role is used more than once on a page, the `aria-label` attribute should also be used in order to distinguish between the two regions.

```js
<Container aria-label="Page content pagination">
```

Reply:
Learnt something, thanks !